### PR TITLE
Games

### PR DESCRIPTION
**Pull request by 01000011-00110111**
Merge them as the game is almost finished. And we need to stop working on other features on this branch.

### CODE REVIEW

File: database.py
Comment (01000011-00110111) [inline database.py:1]: This one will be worked on more in a new branch as it will be a long wile before if fully done

File: filtering.py
Comment (01000011-00110111) [inline filtering.py:1]: Look at the roomid at line 217

File: static/games/TowerD.js
Comment (01000011-00110111) [inline static/games/TowerD.js:1]: Remove this file for now or do not add it to the main branch

File: static/styles.js
Comment (01000011-00110111) [inline static/styles.js:1]: This needs to be reworked

File: templates/games/TowerD.html
Comment (01000011-00110111) [inline templates/games/TowerD.html:1]: again don't add to the main branch

File: templates/index.html
Comment (01000011-00110111) [inline templates/index.html:1]: remove the 1.1.2-G beta and change it to 1.1.2

**Review by 01000011-00110111 — COMMENTED** [review]
once we fix the stuff I commented on it will be okay to push to main.

File: filtering.py [hunk 1]
```diff
@@ -167,7 +214,7 @@ def find_cmds(message, user, roomid):
     origin_room = None
     match = re.findall(r"\(|\)", command_string)
 
-    if perms == 'dev' and roomid == 'jN7Ht3giH9EDBvpvnqRB' and match != []:
+    if perms == 'dev' and roomid == 'jN7Ht3giH9EDBvpvnqRB12' and match != []:
```
Comment (cserver45) [inline filtering.py:217]: should change this on the README as well

**Review by cserver45 — CHANGES_REQUESTED** [review]
Fix some of the things listed, some are just comments for later

File: templates/settings.html [hunk 2]
```diff
@@ -104,7 +107,7 @@ <h3>Extras</h3>
 {% if error %}
     <h2 style="color:red; pointer-events: none;">{{error}}</h2>
 {% endif %}
-<h6><a href="https://github.com/01000011-00110111/School-Chat">Source Code</a></h6>
+<h6><a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ">Source Code</a></h6>
```
[the remark this answers is not in the document]
Comment (01000011-00110111) [reply templates/settings.html:110]: who did that

[the remark this answers is not in the document]
Comment (cserver45) [reply templates/settings.html:110]: idk, see which commit it got changed on

[the remark this answers is not in the document]
Comment (01000011-00110111) [reply templates/settings.html:110]: FIxting that now

File: templates/settings.html [hunk 3]
```diff
@@ -67,6 +67,9 @@ <h3>Account</h3>
         <div class="themeContent">
             <a class="themeOption" id='dark' onclick="setTheme('dark');">Dark</a>
             <a class="themeOption" id='light' onclick="setTheme('light');">Light</a>
+            <a class="themeOption" id='holiday' onclick="setTheme('holiday');">H🎃liday</a>>
+         
+          
             <!-- <a class="themeOption" onclick="setTheme('special');">Special</a> -->
```
[the remark this answers is not in the document]
Comment (01000011-00110111) [reply templates/settings.html:73]: not sure yet

File: templates/games/TowerD.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
+{% extends 'index.html' %}
+{% block header %}
+<meta charset="UTF-8">
+<title>Chat Games</title>
+<script src="static/games/TowerD.js"></script>
+<link rel="stylesheet" href="static/games/games.css"> 
+{% endblock %}
+{% block content %}
+<div id="topScores" style="position: absolute; left: 10px; top: 50px; color: #FFF; font-size: 18px;">
```
[the remark this answers is not in the document]
Comment (01000011-00110111) [reply templates/games/TowerD.html:9]: We are removing this game

File: filtering.py (hunk 1 above)
Comment (01000011-00110111) [reply filtering.py:217]: Its a typo

**Review by 01000011-00110111 — COMMENTED** [review]
Going well

**Comment by 01000011-00110111** [comment]
We will add it but not sure if the game should stay yet

**Comment by 01000011-00110111** [comment]
if you want we can get rid of the games

**Pull request merged**